test/docker-e2e/go.mod

```diff
@@ -77,6 +77,7 @@ require (
 	github.com/bits-and-blooms/bitset v1.17.0 // indirect
 	github.com/bytedance/sonic v1.14.0 // indirect
 	github.com/bytedance/sonic/loader v0.3.0 // indirect
+	github.com/celestiaorg/go-square/v2 v2.3.3-rc0 // indirect
```

Comment:
We can probably create v2.3.3 official release now

Reply:
https://github.com/celestiaorg/celestia-app/pull/5781/commits/a411a9ac36d99db7288a71d94dd06dbeb26f0037